### DISCUSSION TITLE
Research ideas

### DISCUSSION THREAD

**Discussion by zedomel** [globalbioticinteractions/prestonocene#2] [Ideas]
# A place to discuss and share research ideas.

**Comment by seltmann** [comment]
It would be interesting to understand trends regarding who (and for what taxa) biotic interaction information is captured. I assume that it has historically been important in some groups (bees, plant bugs, virus, plants). We could calculate the percentage of the entire order (species, # records) and Shannon's diversity index for those that do or do not have biotic interaction information.

**Reply by seltmann** [reply]
Same could be said for collectors. I know that particular collectors are quite focused on biotic interaction data.

**Comment by seltmann** [comment]
Historic trends such as which are the first biotic interaction records, and have they increased or decreased over time.

**Comment by seltmann** [comment]
Different taxon-based communities likely describe the interactions in different ways. Can we examine the language used (interaction types) across communities. Are there similarities. Perhaps some cluster analysis would be appropriate here.

**Comment by seltmann** [comment]
Ok, so this one is a bit wild, but I wonder if we incorporate geography could we create a map of interaction data richness? One would expect that places that have more species would have larger, more connected networks. Does this assumption hold up for biotic data? We could get the species richness information from GBIF and put the networks together from GloBI.

**Reply by seltmann** [reply]
We could include an interaction accumulation curve for each geographic area

**Reply by seltmann** [reply]
https://r-spatialecology.github.io/landscapemetrics/

**Comment by zedomel** [comment]
- Summary statistics:  
   - Most recorded groups (which group of organism have a greater % of interactions?)
   - Kingdom down to order/family
- Temporal distribution and trends.
- Geographical distribution and trends.
- Shannon diversity index vs. species richness (after some select some location)
- Knowledge gaps
- Try to identify species names from data ML algorithm [https://github.com/dpl10/quaesitor](https://github.com/dpl10/quaesitor) and compare to a manual classification of a subset of the records
